### DISCUSSION TITLE
AI Thinker audio

### DISCUSSION THREAD

**Discussion by al-davis** [schreibfaul1/ESP32-audioI2S#711] [General]
In the repository I see reference to the AIThinker audio board.  I discover that it's old and discontinued but that clones are available.  Seems like a reasonable way to go but also see no support in the Arduino IDE  for either this board or the special ESP32 on it.  Is it a reasonable board to use ?  Can I get the Arduino IDE to support it ?  Does VScode support it ?

**Comment by schreibfaul1** [comment]
The AIThinker board is not bad. The disadvantage is the few free GPIOs for your own extensions. Have a look here:https://www.mikrocontroller.net/topic/474383#new

**Comment by al-davis** [comment]
Thank you, wealth of info in that thread.  Saw that I just use ESP32 dev mod as the device type.  Still working my way thru the thread, I require translation.

**Reply by Marco1971Repo** [reply]
I'm using the board, this library togheter this [https://github.com/pschatzmann/arduino-audio-driver](url) and it work like a charm.
